articles/synapse_docker_registry.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Synapse Docker Registry
+layout: article
+excerpt: The Synapse Docker registry provides a space for synapse users to store and distribute their Docker images per Synapse project.  The Docker images will inherit the some access control settings as the project, so users can choose to develop the tool in private then share it after it is complete.
+---
+
+##Synapse Docker Registry
+Docker containers wrap a piece of software in a complete filesystem that contains everything needed to run.  This can be extremely helpful as software can have many dependencies, so by installing all of them in a container, users can avoid going through the trouble of installing the software on their own computer.  These Docker images can then be stored and distributed on a Docker registry.  The Synapse Docker registry will allow users to create software on a per project basis which can be easily shared across synapse. To learn more about [Docker](https://www.docker.com/products/overview) and [Docker registry](https://www.docker.com/products/docker-registry)
+
+
+### Creating a new Docker image
+Lets begin by creating a custom docker image.  Users can choose to either modify an existing docker image or build a docker image from a Dockerfile.  Docker images must be tagged with 'docker.synapse.org/synapseProjectId/myreponame' to allow images to be saved. 
+
+**Tagging an existing docker image to save onto the synapse registry**
+
+```
+docker pull ubuntu
+```
+To tag an existing docker image, users must know the IMAGE ID.  The IMAGE ID can be found by doing:
+```
+docker images
+#REPOSITORY	TAG	IMAGE ID	CREATED	SIZE
+#ubuntu	latest	f8d79ba03c00	6 days ago	126.4 MB
+```
+Tag the docker image:
+```
+docker tag f8d79ba03c00 docker.synapse.org/syn12345/mytestrepo:version1 
+#syntax: docker.synapse.org/<projectId>/<repoName>:<tag>
+```
+You can also choose to not tag your image with anything, which will by default tag it with `latest`.
```

Comment:
"You can also choose to not tag your image with anything" -> "You can also choose to not tag your image with an explicit tag"